dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -295,7 +295,7 @@ export const NewRunsRow = (props) => {
             onDateSelect={props.onDateSelect}
           />
         ) : (
-          formatDateTime(item.metadata.server.deletion)
+          item.metadata.server.deletion
```

Comment:
Augh!  Apparently I failed to commit my "second thought" on my previous review!  😬  My apologies, Varshini!!

My concern here is that the Dashboard should not be relying on the Server to provide the deletion date in the format which the Dashboard requires.  That is, we should be separating "representation" from "presentation":  the Dashboard should be taking the value provided by the Server (which is a "date" of some sort) and doing its own formatting of it.

So, I suggest creating a new function, e.g.,
```javascript
export const formatDate = (dateString) => {
  const dateObj = new Date(dateString + "Z");    // Make sure it's in UTC
  const formObj = new Intl.DateTimeFormat("en-US", {dateStyle: "medium"})
  return formObj.format(dateObj);
};
```
and then calling that here (and at line 376) instead of `formatDateTime()`.

Reply:
Whoa there: `datestring + "Z"` is very definitely "relying on the Server date representation". There has to be a better way to force the `Date` object to UTC.

Secondarily ... `server.deletion` is a *date*, not a *date-time*. Yeah, for display formatting we need to force it to UTC because we don't want to show the wrong localized date; but showing just the normalized date would be better.

It seems Varshini's code avoids this complication by just displaying the `server.deletion` value directly. Given the inexact nature of the value (we don't guarantee exactly when or if the server will make a deletion pass), I think that's fine for now.

Reply:
> Whoa there: `datestring + "Z"` is very definitely "relying on the Server date representation".

I'm not sure that there is any problem with relying on the Server's representation.  What I objected to is presenting the raw value from the Server directly to the user -- I wanted that presentation separated from the original representation.

All that the code adding the `Z` depends on is that the string from the Server lacks a time zone designator; it'll work with basically anything which looks like a date or time -- it just ensures that Javascript interprets it as UTC.

> showing just the normalized date would be better.

That's exactly what (I believe) my suggested code _does_.  (That is, even if the `dateString` includes time information, it will still return only the date portion.)

> It seems Varshini's code avoids this complication

Yeah...it reports what the Server said...and, as long as we stick with that string and never hint to Javascript that it's actually a date, we'll be fine...but, it would be better if the Dashboard owned the presentation.